Code/Engine/RendererDX11/CommandEncoder/CommandEncoderImplDX11.h

```diff
@@ -112,15 +112,19 @@ class EZ_RENDERERDX11_DLL ezGALCommandEncoderImplDX11 final : public ezGALComman
 
   virtual void SetIndexBufferPlatform(const ezGALBuffer* pIndexBuffer) override;
   virtual void SetVertexBufferPlatform(ezUInt32 uiSlot, const ezGALBuffer* pVertexBuffer, ezUInt32 uiOffset) override;
-  virtual void SetVertexDeclarationPlatform(const ezGALVertexDeclaration* pVertexDeclaration) override;
-  virtual void SetPrimitiveTopologyPlatform(ezGALPrimitiveTopology::Enum topology) override;
 
-  virtual void SetBlendStatePlatform(const ezGALBlendState* pBlendState, const ezColor& blendFactor, ezUInt32 uiSampleMask) override;
-  virtual void SetDepthStencilStatePlatform(const ezGALDepthStencilState* pDepthStencilState, ezUInt8 uiStencilRefValue) override;
-  virtual void SetRasterizerStatePlatform(const ezGALRasterizerState* pRasterizerState) override;
+  virtual void SetGraphicsPipelinePlatform(const ezGALGraphicsPipeline* pGraphicsPipeline) override;
+  virtual void SetComputePipelinePlatform(const ezGALComputePipeline* pComputePipeline) override;
+
+  void SetVertexDeclarationPlatform(const ezGALVertexDeclaration* pVertexDeclaration);
```

Comment:
Can we make these functions private now?

Reply:
Of course! I'll also remove the Platform at the end so it's clear it's no longer part of the interface.